### ISSUE TITLE
New channel added not appearing in Admin page list of content or on Learn

### ISSUE THREAD

**Issue by laurenlichtman** [learningequality/kolibri#4117]
### Observed behavior
It seems that when updating a channel, the channel itself gets set as unavailable, and is absent from both learn and the device channel page.

This occurred once when I was importing content locally, and another time when I was importing content from Studio. When I tried importing again, it worked in the future.

### Expected behavior
You expect for a channel to appear once it has been loaded.

### User-facing consequences
It's highly confusing, and also requires work to be re-done (and in lower bandwidth 
…

### Context
Tell us about your environment, including:
 * v0.10
 * Windows
 * Chrome
…

**Comment by rtibbles** [comment]
@mrpau-julius could you see if you can replicate this bug on latest 0.10?

**Comment by rtibbles** [comment]
Additional report from UNETE: " i'm browsing Kolibri platform and found some bug, but maybe is just a usability thing. Some days ago i downloaded Open Stax channel (partially). Today i try the feature "importar más" inside the channel and get an "Actualizar" blue button... so i click on it hoping that the contents that I had were updated.. instead they delete... if i have to re-select the partial contents that i download in the past, is probably to forget which one pick last time"

**Comment by mrpau-julius** [comment]
@laurenlichtman in what channel you import to kolibri v0.10? So that it's easily to replicate the bug.

**Comment by mrpau-julius** [comment]
Now I trace this issue, yes It's has a bug when you update your channel in kolibri studio(https://studio.learningequality.org).
step to reproduce 
1. I create a channel (ex. bible study)  in kolibri studio with two .pdf and publish it.
2. I copy the channel ID and paste it to "try adding your token" (kolibri v0.10.1) 
3. I imported the channel that I made and see the channel (ex. bible study)
4. I go back to kolibri studio and add new one  .pdf and publish it.
5. Then I go back again in kolibri import channel (kollibri 0.10.1)  select the channel that I created (ex. bible study) and click update.
6. after the update was completed, I click the X button then the channel that I updated will gone.
